api/src/main/java/com/datastrato/gravitino/authorization/SecurableObjects.java

```diff
@@ -173,9 +176,21 @@ public Type type() {
       return type;
     }
 
+    @Override
+    public List<Privilege> privileges() {
+      return privileges;
+    }
+
+    @Override
+    public void bindPrivileges(List<Privilege> privileges) {
+      Preconditions.checkArgument(
+          privileges != null && !privileges.isEmpty(), "Privileges can't be null or empty");
+      this.privileges = ImmutableList.copyOf(privileges);
+    }
```

Comment:
a) Do you need to check, merge or reject the privileges when bind to this securable object?
b) Do you support call `bindPrivileges` more than once, from your current code, the latter will overwrite the former. I would sugges to support call `bindPrivileges` multiple times.

Reply:
`bindPrivileges` should override the former.
`addPrivileges` should merge with origin privilges.
I prefer providing bindPrivileges first.